client/time/time-ago.ts

```diff
@@ -0,0 +1,48 @@
+const MINUTE = 60 // in seconds
+const HOUR = MINUTE * 60
+const DAY = HOUR * 24
+const WEEK = DAY * 7
+const MONTH = DAY * 30.42
+const YEAR = MONTH * 12
+
+/**
+ * A function which takes the number of seconds representing a difference between two dates, and
```

Comment:
nit: avoid "A function that Xes" type comments in favor of "Xes ...."

e.g. this would be better written as:
"Returns a short string representing the difference between two dates."

I think most of the comment gets too much into the weeds of the actual implementation and can be left out, or replaced with a simple example with output commented